LPD-65635

brianchandotcom opened the pull request.

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

brianchandotcom commented:
See https://github.com/brianchandotcom/liferay-portal/pull/166428

brianchandotcom commented:
@mxs2 you also have baseline errors.

```
	modified:   portal-kernel/src/com/liferay/portal/kernel/service/packageinfo
	modified:   portal-kernel/src/com/liferay/portal/kernel/service/persistence/packageinfo


```
